pyrate/core/gamma.py

```diff
@@ -136,14 +207,39 @@ def parse_dem_header(path):
     return subset
 
 
+def parse_baseline_header(path):
+    """
+    Returns dictionary of DEM metadata required for PyRate
+
+    :param str path: `Full path to Gamma *dem.par file`
```

Comment:
fix up doco for baseline, not DEM

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee